src/aura/ongoingProjectItemDetails/ongoingProjectItemDetails.cmp

```diff
@@ -0,0 +1,109 @@
+<aura:component >
+	<aura:attribute name="projectFilterItem" Type="Project__c"/>
+    <div class="demo-only" style="height: 640px; display:none;">
+      <section role="dialog" tabindex="-1" aria-labelledby="modal-heading-01" aria-modal="true" aria-describedby="modal-content-id-1" class="slds-modal slds-fade-in-open"
+               style="
```

Comment:
why is this style inline? Please define a local class in this component and add this styling, then apply said class to the section tag